src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -586,23 +579,57 @@ def utask_main(uworker_input):
     testcase.put()
 
     if check_deadline_exceeded_and_store_partial_minimized_testcase(
-        deadline, uworker_input.testcase_id, uworker_input.job_type,
-        input_directory, file_list, data, testcase_file_path):
-      return
+        deadline, testcase_id, uworker_input.job_type, input_directory,
+        file_list, data, testcase_file_path):
+      return None
 
   command = testcase_manager.get_command_line_for_application(
       testcase_file_path, app_args=app_arguments, needs_http=testcase.http_flag)
   last_crash_result = test_runner.last_failing_result
 
-  store_minimized_testcase(testcase, input_directory, file_list, data,
-                           testcase_file_path)
-  finalize_testcase(
-      uworker_input.testcase_id,
-      command,
-      last_crash_result,
+  # TODO(pgrace): This is a temporary bookmark comment
+
+  return uworker_io.UworkerOutput(
+      testcase=testcase,
+      input_directory=input_directory,
+      file_list=file_list,
+      data=data,
+      testcase_file_path=testcase_file_path,
+      command=command,
+      last_crash_result=last_crash_result,
       flaky_stack=flaky_stack)
 
 
+def utask_postprocess(output):
+  store_minimized_testcase(output.testcase, output.input_directory,
+                           output.file_list, output.data,
+                           output.testcase_file_path)
+  finalize_testcase(
+      output.testcase,
+      output.command,
+      output.last_crash_result,
+      flaky_stack=output.flaky_stack)
+  del output
```

Comment:
nit: not necessary anymore. Was only used to avoid linting errors about not using an argument.